src/components/Toolbelt/Toolbelt.js

```diff
@@ -6,72 +6,63 @@ import classNames from 'classnames'
 
 import FarmhandContext from '../../Farmhand.context'
 import './Toolbelt.sass'
-import { fieldMode } from '../../enums'
-import { tools, pixel } from '../../img'
-
-const { CLEANUP, HARVEST, WATER } = fieldMode
+import { tools as toolImages, pixel } from '../../img'
+import tools from '../../data/tools'
 
 export const Toolbelt = ({
   fieldMode: currentFieldMode,
   handleFieldModeSelect,
-}) => (
-  <div className="Toolbelt">
-    <div className="button-array">
-      {[
-        {
-          alt: 'A watering can for hydrating plants.',
-          fieldMode: WATER,
-          toolImageId: 'watering-can',
-        },
-        {
-          alt: 'A scythe for crop harvesting.',
-          fieldMode: HARVEST,
-          toolImageId: 'scythe',
-        },
-        {
-          alt:
-            'A hoe for removing crops and disposing of them. Also returns replantable items to your inventory.',
-          fieldMode: CLEANUP,
-          toolImageId: 'hoe',
-        },
-      ].map(({ alt, fieldMode, toolImageId }, i) => (
-        <Tooltip
+}) => {
+  let i = 0
+  let toolbeltIcons = []
+
+  for (const [toolId, tool] of Object.entries(tools)) {
+    const { alt, fieldMode } = tool
+    i += 1
+
+    toolbeltIcons.push(
+      <Tooltip
+        {...{
+          key: fieldMode,
+          placement: 'top',
+          title: (
+            <>
+              <p>{alt}</p>
+              <p>(shift + {i})</p>
```

Comment:
Just to make things easy for you: You can wire up this keybinding for the Shovel by updating these maps: 

https://github.com/jeremyckahn/farmhand/blob/aff10c9069251c5d01b8f35ae10bb5fdf8a448e5/src/components/Field/Field.js#L39-L43

https://github.com/jeremyckahn/farmhand/blob/aff10c9069251c5d01b8f35ae10bb5fdf8a448e5/src/Farmhand.js#L436-L438

Reply:
thanks for the pointer!

Reply:
what do you think about moving the hotkey to the 'tools' data obj as well? i felt like it might also need an 'order' for the order to display them.

Reply:
right now these strings you pointed out rely on the order of the other loop, which i could see easily breaking

Reply:
Yeah that's fine with me. It'd be nice for all of this tightly-coupled logic to be in one place!